src/api/geofirestore.js

```diff
@@ -0,0 +1,57 @@
+import * as firebase from 'firebase';
+import * as geofirestore from 'geofirestore';
+import PropTypes from 'prop-types';
+import firebaseConfig from '../../config/firebase.config.js';
+
+const BIKE_COLLECTION_NAME = 'bikes';
+
+// Example usage
+/*
+const demoBike1 = {
+name: 'DELETEME',
+frame: 'medium',
+style: 'mountain',
+coordinates: new firebase.firestore.GeoPoint(37.77825, 122.4224),
+keywords: 'air-fork,light',
+pic_url: 'https://firebasestorage.googleapis.com/v0/b/bikekollective-434ce.appspot.com/o/IMG_20200315_121358979.jpg?alt=media&token=f343ff6a-cb83-41ab-a63d-8aa94a3eb2fd'
+}
+
+const geofirestore = getGeoStore();
+saveBike(geofirestore, demoBike1);
+*/
+
+// Returns a new geofirstore
+// TODO - put this in context?
+export function getGeoStore() {
+    if (!firebase.apps.length) {
+        firebase.initializeApp(firebaseConfig);
+    }
+    const firestore = firebase.firestore();
+    return geofirestore.initializeApp(firestore);
+}
+
+// Saves a bike to the geostore
+export async function saveBike(geoFirestore, bike) {
```

Comment:
This should be updated to use a bike object...

Reply:
Actually, I think this is OK as it is, since just saving a bike to the datastore does not require a lot of the bike member variables, like id, geohash, etc.